how-tos/relpro.md

```diff
@@ -156,50 +156,87 @@ $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.
 $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.ini --branch-and-product-config=/home/cltbld/releasetasks/releasetasks/release_configs/prod_mozilla-esr52_firefox_rc_graph_2.yml --common-task-id=$TASK_TASKID_FROM_GRAPH1
 ```
 
+## 3. signoffs
```

Comment:
up until this PR, this section has signified how to do an action[0] that is being tracked in releasewarrior. Since, I'm assuming, you are not creating an associated action in warrior release templates because its state is already being tracked in Balrog, I wonder if this "how to do sign-offs" is more suited under the per-requisite section like 'how to use tc-talker'[1]?

It doesn't really matter and I'm fine with it as is if you think it makes sense but I feel like this is more of a tutorial for doing signoffs in balrog and less about **when** to do so in a particular release.

[0] https://github.com/mozbhearsum/releasewarrior/blob/multiple-signoffs
/how-tos/relpro.md#actions
[1] https://github.com/mozbhearsum/releasewarrior/blob/multiple-signoffs/how-tos/relpro.md#tctalker-setup